.circleci/config.yml

```diff
@@ -264,12 +269,13 @@ workflows:
           filters:
             tags:
               only: /.*/
-      - build:
-          name: build-nodejs-experimental
-          nodejs: *nodejs_experimental
-          filters:
-            tags:
-              only: /.*/
+      # TODO(https://github.com/mozilla/web-ext/issues/3015): re-enable this once fixed
+      # - build:
+      #     name: build-nodejs-experimental
+      #     nodejs: *nodejs_experimental
+      #     filters:
+      #       tags:
+      #         only: /.*/
```

Comment:
Instead of commenting out everywhere, set the experimental version to the "next" version so that there is a single comment a single line to change later.

Reply:
I oped for another approach, instead of running the job for next twice, which would be a waste, I applied some tweaks to keep the job but allow it to fail (at least that would let me see if anything changes over the time without trying that out locally first, e.g. in a testdouble deps update PR I could just check the circle ci logs to confirm if the dependency update fixed the issue with nodejs 21).

Reply:
the approach I described in https://github.com/mozilla/web-ext/pull/3014#discussion_r1452546949, and so I'm marking this comment thread as resolved (but let me know if there are some followups comments on this site that would need further discussion).